x/mint/types/keys.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+// KeyMinter is the key to use for the Minter in the mint store.
+var KeyMinter = []byte("Minter")
```

Comment:
the store keys prefixes seems different from the sdk, is this expected?

https://github.com/cosmos/cosmos-sdk/blob/0e34478eb7420b69869ed50f129fc274a97a9b06/x/mint/types/keys.go#L5
```go 

var (
	// MinterKey is the key to use for the keeper store.
	MinterKey = collections.NewPrefix(0)
	ParamsKey = collections.NewPrefix(1)
)

const (
	// module name
	ModuleName = "mint"

	// StoreKey is the default store key for mint
	StoreKey = ModuleName
)

```

Reply:
Good point! No they should be same and let me fix it